frontend/src/components/NewHome/components/TermsConditions.js

```diff
@@ -0,0 +1,36 @@
+import { Typography } from '@mui/material';
+import { Container } from '@mui/system';
+import React, { useState } from 'react';
+import CustomSnackbar from '../../Layouts/Snackbar';
+
+const TermsConditions = () => {
+  const [snackbarOpen, setSnackbarOpen] = useState(false);
+  const [snackbarSeverity, setSnackbarSeverity] = useState("success");
+  const [snackbarMessage, setSnackbarMessage] = useState("Hello, I'm the Terms & Conditions page.");
```

Comment:
Move the Snackbar component to the Hero page. The Terms-Condition component should only contain the return() statement